anthos-bm-apigee/docs/quickstart.md

```diff
@@ -0,0 +1,590 @@
+## Quick starter
+
+### Setup the bare metal infrastructure
+
+
+1. Create a Service Aaccount with Owner Role and download the key file. Activate the Service Account. If you have executed ./resources/run_prerequisite.sh , verify that you have the key file downloaded and service account is created with right permission.
```

Comment:
Can we just have one option here?

Either run the `./resources/run_prerequisite.sh` script or just show how to create the key. Also if there is `Owner` role do we need the `Apigee.Admin` role?

Also since we have mentioned this as a pre-requisite I dont think we need this step here.

Reply:
Removed.